packages/graphql-modules/package.json

```diff
@@ -43,13 +48,14 @@
     "ramda": "^0.29.0"
   },
   "publishConfig": {
-    "access": "public",
-    "directory": "dist"
+    "directory": "dist",
+    "access": "public"
   },
   "buildOptions": {
     "copy": [
       "LICENSE.md",
       "README.md"
     ]
-  }
+  },
+  "type": "module"
```

Comment:
This is the main reason I've considered a major bump.

It might impact in some projects